pypeit/calibrations.py

```diff
@@ -652,23 +652,23 @@ def get_wv_calib(self):
         # TODO : Do this internally when we have a wv_calib DataContainer
         binspec, binspat = parse.parse_binning(self.msarc.detector.binning)
 
-        # Instantiate
-        self.waveCalib = wavecalib.WaveCalib(self.msarc, self.slits, self.spectrograph,
-                                             self.par['wavelengths'], binspectral=binspec,
-                                             det=self.det,
-                                             master_key=self.master_key_dict['arc'],  # For QA naming
-                                             qa_path=self.qa_path, msbpm=self.msbpm)
-        # Load from disk (MasterFrame)?
-        masterframe_name = masterframe.construct_file_name(wavecalib.WaveCalib, self.master_key_dict['arc'],
+        masterframe_name = masterframe.construct_file_name(wavecalib.WaveCalib,
+                                                           self.master_key_dict['arc'],
                                                            master_dir=self.master_dir)
         if os.path.isfile(masterframe_name) and self.reuse_masters:
-            # Load from disk
-            self.wv_calib = self.waveCalib.load(masterframe_name)
+            self.wv_calib = wavecalib.WaveCalib.from_file(masterframe_name)
+            self.wv_calib.is_synced(self.slits)
```

Comment:
Under what circumstances would the slits in wavecalib ever not be aligned with the slits in edgetrace? I don't understand why this functionality is necessary and why the code would not just fault if it were not satisfied since it would mean something fishy is going on.

Reply:
renamed to `chk_synced`